test/unit/methods.spec.ts

```diff
@@ -1114,10 +1114,10 @@ describe("Methods call", () => {
       // Given
       const givenLastIndex = 5;
       const flicking = flickingInfo.instance;
-      flicking.setLastIndex(givenLastIndex);
 
       // When
-      await simulate(flickingInfo.element, { deltaX: -1000, duration: 50 });
+      flicking.setLastIndex(-1);
```

Comment:
@WoodNeck What is the purpose of 1) setting `-1` and 2) moving setLastIndex order (after simulate)?

Reply:
As setLastIndex calls resize(), then checks `needPanel` events should be triggered.
Previous test couldn't be passed as needPanel is triggered while calling `setLastIndex`, not simulating.
So, I've managed to fix it to call `setLastIndex(-1)` first, which removes all panels.
Then I call `setLastIndex()` again to trigger `needPanel`.

Reply:
Thank you for your kindness.